packages/foundry/package.json

```diff
@@ -3,19 +3,22 @@
   "version": "0.0.1",
   "type": "module",
   "scripts": {
-    "account": "node scripts-js/ListAccount.js",
-    "chain": "anvil --config-out localhost.json",
+    "verify-keystore": "cast wallet address 2>/dev/null ||  echo 'Keystore verified' && exit 1",
```

Comment:
if there's an error in `verify-keystore` script, it echoes 'Keystore verified', so 'Keystore' is `process.argv[2]`. And terminal shows this

```
yarn account                                           
🔍 Checking account: Keystore
🚫️ Invalid account, please run `yarn account:generate` or `yarn account:import`
```

Maybe allow it to just throw an error? Or show `🔍 Checking account` log only if account is valid

Reply:
yes, if it is successfull it will go through, the problem is that this is an interactive setup, for now is the best solution i could get. it will show you an error if it fails and a good logging telling you what to do `yarn account:generate` . if the echo is not used, the next command wont run

Reply:
i have updated this script, it should be better now @rin-st

Reply:
Thanks! Now it doesn't log `🔍 Checking account` line on error